manifests/init.pp

```diff
@@ -497,7 +497,7 @@
   Variant[Undef,Integer,Pattern[/^(\d+)$/]] $rabbitmq_port = undef,
   Optional[String]   $rabbitmq_host = undef,
   Optional[String]   $rabbitmq_user = undef,
-  Optional[String]   $rabbitmq_password = undef,
+  Variant[Undef,Sensitive,String] $rabbitmq_password = undef,
```

Comment:
@ghoneycutt about the diff, I actually talked based on the wrong assumption that we were managing the file content via the file type and not via custom types.